.github/pull_request_template.md

```diff
@@ -48,11 +57,4 @@ This PR has:
 - [ ] added integration tests.
 - [ ] been tested in a test Druid cluster.
 
-<!-- Check the items by putting "x" in the brackets for the done things. Not all of these items apply to every PR. Remove the items which are not done or not relevant to the PR. None of the items from the checklist above are strictly necessary, but it would be very helpful if you at least self-review the PR. -->
-
-<hr>
-
-##### Key changed/added classes in this PR
- * `MyFoo`
- * `OurBar`
- * `TheirBaz`
+<!-- Once you open a PR, please _avoid using force pushes and rebasing_ since these make it difficult for reviewers to see what you've changed in response to their reviews. See [the contributing doc](https://github.com/apache/druid/blob/master/CONTRIBUTING.md#if-your-pull-request-shows-conflicts-with-master) for more details. -->
```

Comment:
do we want to consider raising this comment up to the top of the template in the hopes of more people reading it vs it being at the bottom and possibly being missed/ignored?

Reply:
Sounds good. I moved it to the top.